distribution/src/bin/elasticsearch-keystore.bat

```diff
@@ -4,9 +4,9 @@ setlocal enabledelayedexpansion
 setlocal enableextensions
 
 set ES_MAIN_CLASS=org.elasticsearch.common.settings.KeyStoreCli
-call "%~dp0elasticsearch-cli.bat" ^
-  %%* ^
-  || exit /b 1
+call "%~dp0elasticsearch-cli.bat" %%* || goto exit
```

Comment:
Would you maintain the line breaks that we have? It keeps these scripts looking the same as the Unix scripts too.